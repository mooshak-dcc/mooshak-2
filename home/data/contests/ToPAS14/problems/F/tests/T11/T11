512 5 9
AFFFAFFFF
AVVVVVXVA
AVVVIVXVA
AVVVVVTVF
AAAAAAAAA
1
S6
