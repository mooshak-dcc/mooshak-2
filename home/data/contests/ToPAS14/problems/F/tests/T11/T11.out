Dentro(1,1):512:ND
